lib/mixlib/install/options.rb

```diff
@@ -22,6 +22,7 @@ module Mixlib
   class Install
     class Options
       class InvalidOptions < ArgumentError; end
+      class EnvironmentVariablesNotFound < StandardError; end
```

Comment:
Let's rename this `ArtifactoryCredentialsNotFound`. I think that is a bit more clear in terms of what the error means.